api/account/models.py

```diff
@@ -158,6 +171,8 @@ class Meta:
         help_text="The use case that the creator of this community (Scorer) would like to cover",
     )
 
+    allo_scorer_id = models.CharField(max_length=42, unique=True, null=True, blank=True)
```

Comment:
let's call this `external_scorer_id` 
I think it is best if it is a generic name

Reply:
Makes sense. I mainly added this because of the requirement that says the description of the created scorer should be `description="Programmatically created by Allo",` I removed all of the references in the code, but left this description